pyaedt/hfss3dlayout.py

```diff
@@ -100,24 +69,56 @@ def get_string(self):
             return None
 
 
-class Hfss3dLayout(FieldAnalysis3DLayout, object):
-    """HFSS 3D Layout Object
+class Hfss3dLayout(FieldAnalysis3DLayout):
+    """HFSS 3D Layout instance interface.
+
+    This class contains all HFSS 3D Layout functionalities. It
+    inherits all objects that belong to HFSS 3D Layout, including EDB
+    API queries.
 
     Parameters
     ----------
     projectname : str
-        Name of the project to select or the full path to the project or AEDTZ archive to open. 
-        If ``None``, try to get the active project and, if none exists, create an empty project.
+        Name of the project to select or the full path to the project
+        or AEDTZ archive to open.  If ``None``, try to get the active
+        project and, if none exists, create an empty project.
     designname : str
-        Name of the design to select. If ``None``, try to get the active design and, if none exists, create an empty design.
+        Name of the design to select. If ``None``, try to get the
+        active design and, if none exists, create an empty design.
     solution_type : str
-        Solution type to apply to the design. If ``None``, use the default.
+        Solution type to apply to the design. If ``None``, use the
+        default.
     setup_name :
-        Name of the setup to use as the nominal. If ``None``, the active setup is used or nothing is used.
+        Name of the setup to use as the nominal. If ``None``, the
+        active setup is used or nothing is used.
+
+    Examples
+    --------
+    Create an ``Hfss3dLayout`` object and connect to an existing HFSS
+    design or create a new HFSS design if one does not exist.
+
+    >>> from pyaedt import Hfss3dLayout
+    >>> aedtapp = Hfss3dLayout()
+
+    Create an ``Hfss3dLayout`` object and link to a project named
+    ``projectname``. If this project does not exist, create one with
+    this name.
+
+    >>> aedtapp = Hfss3dLayout(projectname)
+
+    Create an ``Hfss3dLayout`` object and link to a design named
+    ``designname`` in a project named ``projectname``.
+
+    >>> aedtapp = Hfss3dLayout(projectname,designame)
+
+    Create an ``Hfss3dLayout`` object and open the specified project.
+
+    >>> aedtapp = Hfss3dLayout("myfile.aedt")
 
-    Returns
-    -------
+    Create a ``Desktop on 2021R1`` object and then creates an
```

Comment:
I've never seen verb used in a Returns section. The first "create" has no s, but the second one does. Assume that you'd want to remove the  s  on the second occurrence, but I'm not sure.

Reply:
Verbs are fine, look at how `pandas` does their examples:
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.copy.html#pandas.DataFrame.copy

Agree that we don't need a "s" in the second "create"

Reply:
```suggestion
    Create a ``Desktop on 2021R1`` object and then create a
```